src/snowflake/snowpark/modin/plugin/extensions/base_overrides.py

```diff
@@ -2262,7 +2262,7 @@ def _get_index(self):
         return self._query_compiler.index
 
     idx = Index(query_compiler=self._query_compiler)
-    idx._set_parent(self)
+    idx._parent.set_parent(self)
```

Comment:
can we update `_set_parent` instead?

Reply:
actually do we need that one line wrapper? the current place seems the only place calls it

Reply:
I reverted it to `_set_parent()` since that seems like a "safer" way to set the parent details without exposing too much